src/utils/css3-generator.js

```diff
@@ -47,16 +47,41 @@
 		return 'rotate' + prefix + theta + suffix;
 	};
 	
+	var getSkew = function (x, y, unit) {
+		if (!!!unit) {
+			unit = 'deg';
+		}
+
+		if (unit != 'deg' && unit != 'rad' && unit != 'turn') {
```

Comment:
We should do a strict comparison `!==` 😁

Reply:
fixer